pkg/handlers/v1/notification.go

```diff
@@ -56,6 +59,7 @@ func (h *NotificationHandler) Handle(ctx context.Context) (Output, error) {
 
 	scanNotifications := make([]scanNotification, len(scans))
 	for offset, scan := range scans {
+		scanCompletedTime := time.Now()
```

Comment:
Sorry if this was unclear, but to determine the elapsed time between scan completion and scan event processing, we actually need to use the incoming `domain.CompletedScan.Timestamp` as the starting point.

That way, we're including the time between asset scan by Nexpose and the time the scan notification arrived in our service. Make sense?

task: use `scan.Timestamp` instead of `time.Now()` for the `scanCompletedTime`